lighthouserc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  ci: {
+    collect: {
+      staticDistDir: "./public",
+      url: [
+        "/",
+        "/about/",
+        "/issues/",
+        "/issues/1/",
+        "/issues/1/data-beyond-vision/",
+        "/404.html"
+    ]
+    },
+    assert: {
+      assertMatrix: [
+        // homepage/issue detail incorrectly error on contrast of background
+        // text with role=presentation
```

Comment:
I don't quite understand this comment — am I understanding it's wrongly detecting an error and you're configuring it to ignore? If so, might be good to add a little more detail about what the problem is.

Will we be able to turn this off at some point? Or if not, is there a chance we will miss other problems because of this configuration?

Reply:
I should expand this, thanks. Basically the transparent background text that appears on the homepage and issue detail page that's generated from the essay previews was generating contrast warnings, but that text isn't really meant to be read (it's decorative), so I gave it `role=presentation`. Lighthouse (so I guess aXe) still flags it...I thought about filing an issue, but for now the best we can do is ignore contrast errors for those pages specifically (I don't think you can get more granular than that in Lighthouse).